apps/system/js/bluetooth.js

```diff
@@ -194,6 +196,10 @@ var Bluetooth = {
     req.onsuccess = function bt_gotDefaultAdapter(evt) {
       self.defaultAdapter = req.result;
       self.initWithAdapter(self.defaultAdapter);
```

Comment:
in v2 we call it `_adapterAvailableHandler` since it does more than ProfilesStatusChanged,  but I keep v1 not changed. it's fine to rename it to similar name.